src/components/NavGlobal.tsx

```diff
@@ -0,0 +1,50 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { cookies } from 'next/headers';
+
+import cloudGovText from '@/../public/img/logos/cloud-gov-logo-text.svg';
+import cloudGovIcon from '@/../public/img/logos/cloud-gov-icon.svg';
+import dashboardIcon from '@/../public/img/logos/dashboard-icon.svg';
+import cloudPagesIcon from '@/../public/img/logos/cloud-pages-icon.svg';
+
+import { LoginButton } from '@/components/auth/LoginButton';
+import { LogoutButton } from '@/components/auth/LogoutButton';
+
+export function NavGlobal() {
+  const cookieStore = cookies();
+  const authSession = cookieStore.get('authsession');
+
+  return (
+    <div id="nav-global">
+      <div>
+        <div className="nav-global-inner">
+          <Link href="/" className="cloud-logo">
+            <Image
+              id="cloud-gov-icon"
+              unoptimized
+              src={cloudGovIcon}
+              alt="cloud.gov"
+            />
+            <Image
+              id="cloud-gov-logo-text"
+              unoptimized
+              src={cloudGovText}
+              alt="cloud.gov"
+            />
+          </Link>
+          <Link href="/" className="nav-product active">
+            <Image unoptimized src={dashboardIcon} alt="dashboard" />
```

Comment:
`alt` attributes for icons that are also labeled with a text label are redundant and discouraged: https://html.spec.whatwg.org/multipage/images.html#a-short-phrase-or-label-with-an-alternative-graphical-representation:-icons,-logos

better to leave the alt attrs her and line 40 as empty strings :)

Reply:
I was wondering if I should be adding an aria attribute telling screen readers to skip the icons, but removing the alt text is simpler!  Thanks!